packages/ckeditor5-core/src/editor/editorconfig.jsdoc

```diff
@@ -231,7 +250,23 @@
  *
  * Check the {@glink features/ui-language UI language guide} for more information about the localization options and translation process.
  *
- * @member {String|Object} module:core/editor/editorconfig~EditorConfig#language
+ * @member {String|module:core/editor/editorconfig~LanguageConfig} module:core/editor/editorconfig~EditorConfig#language
+ */
+
+/**
+ * Allows to use different language for the editor UI.
+ *
+ * The language codes are defined in the [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) standard.
+ *
+ * @member {String} module:core/editor/editorconfig~LanguageConfig#ui
+ */
+
+/**
+ * Allows to use different language of the editor content.
+ *
+ * The language codes are defined in the [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) standard.
+ *
+ * @member {String} module:core/editor/editorconfig~LanguageConfig#content
```

Comment:
`textFragmentLanguage` is also supporting ISO 639-2 (3 letters) codes. I'm wondering if we should also extend `language.content` and `language.ui` with ISO 639-2 or rather remove this information from `textFragmentLanguage` and treat ISO 639-2 only as deprecated, compatibility support? I have a feeling that it would be nicer to limit supported codes only to ISO 639-1 as they are more common.

Reply:
I've simplified it to ISO 639-1. It's still possible to use other ISO formats since it's just a string that won't conflict with the current implementation, but using a single format makes our life easier.